config/settings.yml

```diff
@@ -19,13 +19,11 @@ workspace_dir: <%= Rails.root.join('tmp', 'workspace').to_s %>
 sdr:
   local_workspace_root: /dor/workspace
 
-
-# true to post completed submission to Registrar
-dor_submit_ps_xml: false
-ps_reg_endpoint: 'http://registrar.example.edu'
-ps_reg_username: ~
-ps_reg_password: ~
-ps_env: 'test'
+peoplesoft:
+  client_id: ~
+  client_secret: ~
+  base_url: ~
+  token_url: ~
```

Comment:
These are already set up in shared_configs for qa, stage, and uat. Prod to come when UIT has that ready for us, which should be as soon as we successfully test in the other envs.

Reply:
Not that it's holding this up, but should the id/secret at least go into vault and be set via ENV var, and maybe the urls just plain ENV vars?

Reply:
@aaron-collier Yep, I already did that for qa, stage, and uat. Waiting on prod.